test/transforms.jl

```diff
@@ -680,6 +680,49 @@
     @test n1 == n2
   end
 
+  @testset "StdNames" begin
+    # camel, snake, upper tests
+    @test TableTransforms._camel("apple banana") == "AppleBanana"
+    @test TableTransforms._snake("apple banana") == "apple_banana"
+    @test TableTransforms._upper("apple banana") == "APPLEBANANA"
+
+    @test TableTransforms._camel("apple_banana") == "Apple_banana"
```

Comment:
The camel case should remove the underscore and handle the second word with a first capital letter.